### ISSUE TITLE
use pypy if available

### ISSUE THREAD

**Issue by muellermichel** [muellermichel/Hybrid-Fortran#39]
pypy in current measurement (after having optimized all the low handing fruits, i.e. ASUCA is now transpiling in ~2min) is about 30% faster than python. Build system should check for pypy in path and use it if available.

**Comment by szaghi** [comment]
Interesting, 30% is a lot!

Great Michel!

**Comment by muellermichel** [comment]
yeah, actually 30% is just what pypy gets out of it, I already got somewhere between 5x and 10x faster transpilation today (with cpython only) after a few optimizations. But that's nothing to brag about, just a few stupid earlier decisions I had to clean up ;).
